TTS/tts/datasets/preprocess.py

```diff
@@ -37,7 +37,7 @@ def split_dataset(items):
     return items[:eval_split_size], items[eval_split_size:]
 
 
-def load_meta_data(datasets, eval_split=True):
+def load_meta_data(datasets, eval_split=True, ignore_generated_eval=False):
```

Comment:
`eval_split` should already let people choose to generate a eval split or not.  Why someone needs to ignore the eval split.

Reply:
I think as the speaker embeddings will be used for model training, during training you will always need the eval. So I don't see a case where the user doesn't need to extract the embeddings from the eval. But if you still want I can add it as in argparser.

Reply:
I think it is better to add it to argparse and remove this from the function.

Reply:
Ignoring the generated eval is necessary as in some cases some samples are lost. If we don't ignore it in some cases it won't calculate all embeddings causing errors during training. Therefore, if eval_split is not passed in config.json, compute from all and avoid splitting. I cannot remove this until I have a better solution. The current split_dataset loses some samples during the split.

Reply:
I kept the parameter: ignore_generated_eval and added as argparser the eval_split. Please review again.